package/network/utils/iproute2/Makefile

```diff
@@ -100,7 +100,7 @@ endef
 define Package/ss
 $(call Package/iproute2/Default)
   TITLE:=Socket statistics utility
-  DEPENDS:=+libnl-tiny +(PACKAGE_devlink||PACKAGE_rdma):libmnl +kmod-netlink-diag
+  DEPENDS:=+libnl-tiny +PACKAGE_libtirpc:libtirpc +(PACKAGE_devlink||PACKAGE_rdma):libmnl +kmod-netlink-diag
```

Comment:
Why is `+PACKAGE_libtirpc:libtirpc` still needed when you set `HAVE_TIRPC=n`?

Reply:
Did forget to delete it again. Thanks for spotting this.